GOVERNANCE.md

```diff
@@ -81,6 +96,15 @@ Project members may be nominated for the emeritus team through one of the follow
 
 [contributions]: https://github.com/buildpacks/community/blob/main/contributors/guide.md#contributions
 
+## Voting Rules
+
+All voting in the CNB project will adhere to a [lazy consensus](https://openoffice.apache.org/docs/governance/lazyConsensus.html) model. In order for an RFC, proposal, or other motion to pass, the following must hold true:
+
+* There are no votes against the motion
+* There is at least one vote in favor of the motion
+
+The absence of a vote from a party with a binding vote in the process is considered to be a vote in the affirmative. Non-binding votes are welcomed and encouraged.
```

Comment:
Could you clarify what a non-binding vote is in this context, for people who don't know how it works in the general CNCF world?

Reply:
I am working on a thorough write-up of how this works in the context of RFCs - https://github.com/buildpacks/rfcs/pull/218